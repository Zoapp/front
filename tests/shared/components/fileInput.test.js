import React from "react";
import renderer from "react-test-renderer";
import FileInput from "shared/components/fileInput";

describe("FileInput", () => {
  it("renders correctly", () => {
    const component = renderer.create(React.createElement(FileInput));

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("can take an accept prop", () => {
    const component = renderer.create(
      React.createElement(FileInput, { accept: "image/*" }),
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
