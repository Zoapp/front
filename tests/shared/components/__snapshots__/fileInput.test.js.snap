// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FileInput can take an accept prop 1`] = `
<div
  className="fileInput"
>
  <i
    className="material-icons"
  >
    cloud_upload
  </i>
  <input
    accept="image/*"
    onChange={[Function]}
    type="file"
  />
</div>
`;

exports[`FileInput renders correctly 1`] = `
<div
  className="fileInput"
>
  <i
    className="material-icons"
  >
    cloud_upload
  </i>
  <input
    accept={null}
    onChange={[Function]}
    type="file"
  />
</div>
`;
